app/workers/zomato_data_reader.rb

```diff
@@ -0,0 +1,53 @@
+require 'faraday_middleware'
+
+class ZomatoDataReader
+
+  def initialize
+    @connection = Faraday.new(url: ENV['ZOMATO_URL']) do |c|
+      c.request :json
+      c.headers['user-key'] = ENV['ZOMATO_API_KEY']
+      c.adapter Faraday.default_adapter
+    end
+  end
+
+  def fetch_restaurants_list(number_of_restaurants = 2, city_id = 280)
+    response = get_restaurants(number_of_restaurants, city_id)
+    restaurants_info = JSON.parse(response.body)['restaurants']
+    restaurants_info.map {|r| r['restaurant']}
+        .map {|restaurant|
+          {id: restaurant['id'],
+           name: restaurant['name'],
+           cuisine: restaurant['cuisines'].split(',').first,
+           rating: 0,
+           accepts_ten_bis: (rand * 10).to_i < 5,
```

Comment:
Let's go with a more generic solution for randomly choosing true of false, and put it in a separate method.

One way to do it: `[false, true].sample`

Or you can simply default to false, to retain consistency :) .